components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -470,3 +485,91 @@ func checkResultOutputForError(output string) error {
 	}
 	return nil
 }
+
+// Execute custom command in All the node
+/*
+Input: []*NodeObject{
+
+}
+*/
+func ExecuteCmdInAllNodeAndCaptureOutput(nodeObjects []*NodeObject, singleNode bool, outputDirectory string) error {
```

Comment:
move this to `ha_node_operations_utils.go`
Also rename this, with proper explanation name

Reply:
Done